lib/extensions/ar_recent_records.rb

```diff
@@ -0,0 +1,26 @@
+module ArRecentRecords
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    # Returns a hash of counts by date for the :created_on field, with the
+    # date as the key and the count as the value.
+    #
+    # @param date Maximum number of days to look back. Default is 30.
+    # @param group_by Date group to use for collating data. Default is 'day'.
+    # @param filter Additional fields to add to the filter.
+    # @return [Hash] counts per group_by
+    #
+    # Examples:
```

Comment:
Minor, but for example docs, use the `@example` yardoc tag  - https://www.rubydoc.info/gems/yard/file/docs/Tags.md#example

Reply:
Done.